jdenticon/jdenticon-tests.ts

```diff
@@ -0,0 +1,11 @@
+import {jdenticon} from "./index";
```

Comment:
To be honest, I'm not completely sure what the test is supposed to look like.  In my Ionic 2/Angular 2 code that use this Type Definition, I'm using 
```
import jdenticon from "jdenticon";
```
But, then I'm using the following piece in `declarations.d.ts`:

```
declare module 'jdenticon' {
  export interface JdenticonConfig {
    lightness?: {
      color?: number[];
      grayscale?: number[];
    };
    saturation?: number;
  }

  export var jdenticon_config: JdenticonConfig;

  export function drawIcon(ctx: CanvasRenderingContext2D, hash: string, size: number): void;

  export function toSvg(hash: string, size: number, padding?: number): string;

  export function update(el: Element|string, hash?: string, padding?: number): void;

  export let config: JdenticonConfig;

  export const version: string;

}
```
This was based on following the documentation in https://github.com/DefinitelyTyped/DefinitelyTyped#test-a-new-package.  However, I'm not sure about the “leap” in going from `declaration.d.ts` with `declare module …` to `index.d.ts` which doesn't seem to support that declaration statement.  Can you provide any guidance here, so that I don't have to make blind-guessing commits?